donut/modules/newsgroups/routes.py

```diff
@@ -90,7 +90,8 @@ def view_group(group_id):
     if actions['control']:
         applications = helpers.get_applications(group_id)
     messages = None
-    member = groups.is_user_in_group(user_id, group_id)
+    member = auth_utils.is_admin() or groups.is_user_in_group(
```

Comment:
If you change the behavior of `is_admin()` to avoid looking up the username from the user ID, it would be more efficient to pass `user_id` here